api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -64,4 +64,10 @@ message RateLimit {
   // success.
   config.ratelimit.v3.RateLimitServiceConfig rate_limit_service = 7
       [(validate.rules).message = {required: true}];
+
+  // If set to true enables emitting X-RateLimit-* headers by the filter.
+  // The headers are specified in the `draft RFC
+  // <https://tools.ietf.org/id/draft-polli-ratelimit-headers-00.html>`_
+  // Defaults to false.
+  bool enable_response_headers = 8;
```

Comment:
Can we make the name of this field more specific? `enable_x_ratelimit_headers` or similar? It's a little unclear what this does from the field name.